README.md

```diff
@@ -92,31 +92,34 @@ PyMove **proposes**:
 
 ---
 
-## Github installation
-
-1.Clone this repository
-
-`git clone https://github.com/InsightLab/PyMove`
-
-2.Make a branch developer
-
-`git branch developer`
-
-3.Switch to a new branch 
-
-`git checkout developer`
-
-4.Make a pull of branch
+## Creating Virtual Environment
 
-`git pull origin developer`
+It is recommended to create a virtual environment to use pymove. Requirements: Anaconda Python distribution installed and accessible
 
-5.Switch to folder PyMove
+1. In the terminal client enter the following where `yourenvname` is the name you want to call your environment, and replace `x.x` with the Python version you wish to use. (To see a list of available python versions first, type conda search "^python$" and press enter.)
+    - `conda create -n <yourenvname> python=x.x`
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[list-item-content-indent] Don’t use mixed indentation for children, remove 1 space](https://app.codacy.com/gh/InsightLab/PyMove/pullRequest?prid=5506003)